hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/TestHoodieClientMultiWriter.java

```diff
@@ -345,16 +344,16 @@ public void testHoodieClientMultiWriterWithClustering(HoodieTableType tableType)
         numRecords, 200, 2);
     // Start and finish another commit while the previous writer for commit 003 is running
     newCommitTime = "004";
-    SparkRDDWriteClient client2 = getHoodieWriteClient(cfg2);
+    SparkRDDWriteClient client2 = getHoodieWriteClient(cfg);
     JavaRDD<WriteStatus> result2 = updateBatch(cfg2, client2, newCommitTime, "001",
         Option.of(Arrays.asList(commitTimeBetweenPrevAndNew)), "000", numRecords, SparkRDDWriteClient::upsert, false, false,
         numRecords, 200, 2);
     client2.commit(newCommitTime, result2);
     // Schedule and run clustering while previous writer for commit 003 is running
-    SparkRDDWriteClient client3 = getHoodieWriteClient(cfg);
+    SparkRDDWriteClient client3 = getHoodieWriteClient(cfg2);
     // schedule clustering
     Option<String> clusterInstant = client3.scheduleTableService(Option.empty(), TableServiceType.CLUSTER);
-    assertFalse(clusterInstant.isPresent());
+    assertTrue(clusterInstant.isPresent());
```

Comment:
NTR: first two clients makes 2 commits and 1 inflight commit(003). and 3rd client which has inline clustering config enabled, tries to schedule clustering. So, it should be true here. following this, when we try to commit 003, we expect it to fail due to conflicts. So, don't understand why this was set to false before.